pkgs/tools/misc/xvfb-run/default.nix

```
@@ -3,7 +3,7 @@
 let
   xvfb_run = fetchurl {
     url = https://projects.archlinux.org/svntogit/packages.git/plain/trunk/xvfb-run?h=packages/xorg-server;
```

Could you also pin the package so this does not repeat in the future:

```suggestion
    # https://git.archlinux.org/svntogit/packages.git/?h=packages/xorg-server
    url = https://git.archlinux.org/svntogit/packages.git/plain/trunk/xvfb-run?h=packages/xorg-server&id=9cb733cefa92af3fca608fb051d5251160c9bbff;
```